### ISSUE TITLE
search queries don't recognize special chars

### ISSUE THREAD

**Issue by bdionne** [NCIEVS/nci-protege5#6]
This appears to be a protege [issue](https://github.com/protegeproject/lucene-search-tab/issues/1)

**Comment by NCIEVS** [comment]
This was fixed upstream.  Labeling enhancement for v1 release & ready for QA.

**Comment by rmehri** [comment]
I have retested this fix with different properties and searched with special characters and not received any search results "0" match.  The issue description states "search queries don't recognize special chars"
Please let me know your clarification whether we should expect "0" match when special character is used for searching.

![search-special character](https://cloud.githubusercontent.com/assets/8367102/21114223/6a037d88-c07a-11e6-8a0a-c9bb94dfc34d.png)

**Comment by bdionne** [comment]
could it be that nothing matches your search string? 

<img width="1021" alt="screen shot 2016-12-12 at 3 16 32 pm" src="https://cloud.githubusercontent.com/assets/31741/21115168/04f6b118-c07e-11e6-9025-7e8d8e52c83f.png">

**Comment by rmehri** [comment]
Please see attached zip file to play the video for searching special character and return  "0" match.
Server address: https://nciws-d699-v.nci.nih.gov:9090
Used "eric" login

Regards,
Reza

[Search_Special Chr.zip](https://github.com/NCIEVS/nci-protege5/files/648044/Search_Special.Chr.zip)

**Comment by bdionne** [comment]
Ok, I watched this video, which conveys the same information as the single screenshot above.

My question still remains; let me rephrase it another way. Are you sure there is content in your ontology that matches the search string `#$@` ?  In other words maybe `0 matches` is the correct answer?

**Comment by rmehri** [comment]
I have added "$#@" to FULL_SYN property and search result was "$#@" . 
I have also added characters :,!,? to FULL_SYN property and I saved then the search for characters :,!,? and it returned  "$#@".  However, according to the business below, the property cannot include these charters: 
Please see attached snapshots and let me your clarification so I can validate the fix.  

https://github.com/NCIEVS/nci-protege5/wiki/editing-rules-enforced-by-nci-protege

"◦Properties cannot include the characters :,!,? or @ (except for DEFINITION which allows ! and ?)"


![full_sys-special char1](https://cloud.githubusercontent.com/assets/8367102/21148350/e838f632-c125-11e6-9246-702cd81f8cb4.png)

![full_sys-special char](https://cloud.githubusercontent.com/assets/8367102/21148367/f8a44652-c125-11e6-8eba-2fc25d95bb2e.png)
![search](https://cloud.githubusercontent.com/assets/8367102/21148411/27149596-c126-11e6-8b0a-72fbba0a592d.png)

Thanks,
Reza

**Comment by bdionne** [comment]
Good, so it appears searching with special characters is working, which is what this ticket is about. Whether the [editing rules](https://github.com/NCIEVS/nci-protege5/wiki/editing-rules-enforced-by-nci-protege) are being enforced is a separate issue, and should have it's own ticket if there's not one already. 

Note that those [rules](https://github.com/NCIEVS/nci-protege5/wiki/editing-rules-enforced-by-nci-protege) are from the old system and might need reworking for the new version
